_posts/2024-08-19-quarkus-metaprogramming.adoc

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: 'Unleashing the power of metaprogramming with Quarkus extensions'
```

Comment:
Might be interesting to mention serialization and performance in the title since many people might not know what metaprogramming is or might not care if they do because it might be too conceptual for them, whereas if the title mentions serialization gains it might attract more readers?